kolibri/plugins/learn/assets/src/vue/side-nav/index.vue

```diff
@@ -93,6 +126,10 @@
   svg
     fill: $core-action-normal
     transition: fill $core-time ease-out
+    @media screen and (orientation: portrait)
```

Comment:
I wonder if `orientation: portrait` is the most appropriate media query? According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#orientation):

> This value does not correspond to actual device orientation. Opening the soft keyboard on most devices in portrait orientation will cause the viewport to become wider than it is tall, thereby causing the browser to use landscape styles instead of portrait.

Is that the intended behavior?